x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.test.tsx

```diff
@@ -28,21 +28,20 @@ import { TestProviders } from '../../../../common/mock';
 import {
   getRulesEqlSchemaMock,
   getRulesSchemaMock,
```

Comment:
should `components` folder be moved to `rule_execeptions_ui` folder instead of `rule_execeptions`?

Reply:
@vitaliidm I can't say anything about the whole folder. Reusable components that are shared between several different pages (e.g. Exceptions Lists and Rule Details pages) or core subdomains should stay here. Components that are only used on the Exceptions Lists page should be moved to `rule_execeptions_ui` to stay closer to their usage and increase cohesion.